Lib/idlelib/config-extensions.def

```diff
@@ -63,3 +63,13 @@ z-text= Z
 z-in= <Control-Shift-KeyRelease-Insert>
 [ZzDummy_bindings]
 z-out= <Control-Shift-KeyRelease-Delete>
+
+[ParenClose]
+enable = True
+paren_close = True
+tick_close = True
+skip_closures = True
+[ParenClose_cfgBindings]
+p-open = <Key-parenleft> <Key-bracketleft> <Key-braceleft>
+t-open = <Key-'> <Key-">
```

Comment:
I downloaded the PR to take a look at it and couldn't open IDLE because of this error:

`File "/home/cheryl/cpython/Lib/idlelib/editor.py", line 137, in __init__
    self.apply_bindings()
  File "/home/cheryl/cpython/Lib/idlelib/editor.py", line 1071, in apply_bindings
    text.event_add(event, *keylist)
  File "/home/cheryl/cpython/Lib/idlelib/multicall.py", line 374, in event_add
    widget.event_add(self, virtual, seq)
  File "/home/cheryl/cpython/Lib/tkinter/__init__.py", line 1653, in event_add
    self.tk.call(args)
_tkinter.TclError: bad event type or keysym "'"`

I looked at this page for the keysyms:
https://www.tcl.tk/man/tcl8.4/TkCmd/keysyms.htm

and it has `quotedbl                            34     0x0022` and `quoteright                          39     0x0027`, which seems to fix the error.

Also, `'` and `"` aren't available on the config keys screen, so they may need to be added.